code/modules/modular_computers/computers/subtypes/dev_telescreen.dm

```diff
@@ -0,0 +1,70 @@
+/obj/item/modular_computer/telescreen
+	name = "telescreen"
+	desc = "A wall-mounted touchscreen computer."
+	icon = 'icons/obj/modular_telescreen.dmi'
+	icon_state = "telescreen"
+	icon_state_unpowered = "telescreen"
+	icon_state_menu = "menu"
+	hardware_flag = PROGRAM_TELESCREEN
+	anchored = TRUE
+	density = 0
+	base_idle_power_usage = 75
+	base_active_power_usage = 300
+	max_hardware_size = 2
+	steel_sheet_cost = 10
+	light_strength = 4
+	max_damage = 300
+	broken_damage = 150
+	w_class = ITEM_SIZE_HUGE
+
+/obj/item/modular_computer/telescreen/New()
+	..()
+	// Allows us to create "north bump" "south bump" etc. named objects, for more comfortable mapping.
+	name = "telescreen"
```

Comment:
Isn't `name = initial(name)` whould be better and easier to maintain?

Reply:
fixed

Reply:
fixed